pages/index.tsx

```diff
@@ -1,11 +1,51 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
 import { HomeLayout } from "../components/layout";
 import { SubmissionsTable } from "../components/submissions";
+import { Pagination } from "../interfaces";
 
 export default function HomePage() {
+  const [submissions, setSubmissions] = useState([]);
```

Comment:
you're not telling ts what the array is for here. You need to do ```useState<Type[]>([])``` obviously where Type is the submission type, can't remember the exact name of it